tifeatures/dependencies.py

```diff
@@ -50,21 +50,6 @@ def CollectionParams(
     )
 
 
-def OutputType(
```

Comment:
with more output types per endpoint, it's better to handle this in each endpoint